src/components/node/node-page-metadata.tsx

```diff
@@ -0,0 +1,82 @@
+import {getConfigPageField} from "@/lib/gql/fetcher"
+import {
+  MetaTagUnion,
+  MetaTagValue as MetaTagValueType,
+  MetaTagProperty as MetaTagPropertyType,
+  StanfordBasicSiteSetting,
+} from "@/lib/gql/__generated__/drupal.d"
+import {JSX} from "react"
+
+type Props = {
+  /**
+   * Page title without the site name, undefined if the home page.
+   */
+  pageTitle?: string
+  /**
+   * Metatag field data.
+   */
+  metatags?: MetaTagUnion[]
+  /**
+   * If no description metatag is provided by the backend, use this.
+   */
+  backupDescription?: string
+  /**
+   * Additional meta data if desired.
+   */
+  children?: JSX.Element | JSX.Element[]
+}
+
+const NodePageMetadata = async ({pageTitle, metatags, backupDescription, children}: Props) => {
+  const siteName =
+    (await getConfigPageField<StanfordBasicSiteSetting, StanfordBasicSiteSetting["suSiteName"]>(
+      "StanfordBasicSiteSetting",
+      "suSiteName"
+    )) || "Stanford University Libraries"
+
+  const title = pageTitle ? `${pageTitle} | ${siteName}` : siteName
+
+  const hasDescription = metatags?.some(
+    tag => tag.__typename === "MetaTagValue" && tag.attributes.name === "description"
+  )
+
+  return (
+    <>
+      <title>{title}</title>
+      <meta property="og:title" content={title} />
+      <meta name="twitter:title" content={title} />
+
+      {!hasDescription && backupDescription && (
+        <>
+          <meta name="description" content={backupDescription} />
+          <meta name="twitter:description" content={backupDescription} />
+          <meta property="og:description" content={backupDescription} />
+        </>
+      )}
+
```

Comment:
I was testing different content types from the preview site and it looks like I missed the images. Please hold on review, I am working on resolving the issue now. 😮‍💨

Reply:
Whelp, that was an easy fix! 🤦‍♀️